src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-field {
+  margin-top: 20px;
+}
+
+.search-icon-big {
+  position: absolute;
+  margin-top: 25px;
+  margin-left: 25px;
+}
+
+.search-bar {
+  width: 100%;
+
+  box-sizing: border-box;
+
+  border: 1px solid rgb(225, 231, 237);
+  border-radius: 4px;
+
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  font-family: Avenir, serif;
```

Comment:
Move the font-family property to the html tag

Reply:
If I add the font-family property to the html tag, font-family doesn't work. But if through *, it works. What is the problem?